terraspace/README.md

```diff
@@ -13,8 +13,12 @@ To deploy individual stacks:
 
     AWS_PROFILE=<profile> AWS_REGION=<region> TS_ENV=<environment> bundle exec terraspace up shared
     AWS_PROFILE=<profile> AWS_REGION=<region> TS_ENV=<environment> bundle exec terraspace up indexing
-    terraspace up shared   # where shared is app/stacks/shared
-    terraspace up indexing # where indexing is app/stacks/indexing
+    AWS_PROFILE=<profile> AWS_REGION=<region> TS_ENV=<environment> bundle exec terraspace up peer
+    AWS_PROFILE=<profile> AWS_REGION=<region> TS_ENV=<environment> bundle exec terraspace up peer-kubernetes-components
+    terraspace up shared                     # where shared is app/stacks/shared
+    terraspace up indexing                   # where indexing is app/stacks/indexing
+    terraspace up peer                       # where peer is app/stacks/peer
+    terraspace up peer-kubernetes-components # where peer-kubernetes-components is app/stacks/peer-kubernetes-components -var token=$(aws eks get-token --cluster-name <cluster-name>)
```

Comment:
we don't need to specify the `-var token=...` via the command line do we?

Reply:
No, I'm very happy that we don't! :)
I wrote that nightmare there and then forgot to erase it. Done now.